src/Stan/Inspection.hs

```diff
@@ -45,3 +47,21 @@ data Severity
 -- | Show 'Inspection' in a human-friendly format.
 prettyShowInspection :: Inspection -> Text
 prettyShowInspection = show
+
+{- | List of all inspections.
+-}
+inspections :: [Inspection]
+inspections =
+    [ Inspection
+        { inspectionId = Id "HEAD"  -- TODO: we need to come up with naming scheme for inspection ids
+        , inspectionName = "Partial 'head'"  -- TODO: does it make sense?
+        , inspectionDescription = "Usage of partial function 'head' for lists"
+        , inspectionSolution = unlines
+            [ "* Replace list with 'NonEmpty' from 'Data.List.NonEmpty'"
+            , "* Use explicit pattern-matching over lists"
+            ]
+        , inspectionCategory =
+            one $ Category "Partial"  -- TODO: should we convert this to values to avoid typos?
```

Comment:
Yes! That would be nice, and also handy!

Reply:
We also will need a list of all categories

Reply:
@vrom911 Should we move `Category` in a separate module then?

Reply:
Created an issue to do this separately: https://github.com/kowainik/stan/issues/25

Reply:
Sounds good :+1: